File: larissa016/Questao09.py
print("**Quantidade de salgados**")

d = float(input("Informe o total de dinheiro que possui:"))
ca = float(input("Quanto custava o salgado?"))
cg = float(input("Quanto custa um salgado?"))

ca1 = (d)/(ca)
cg1 = (d)/(cg)

print( )
print("Antes com R$" + str(d) + "voce podia comprar" + str(ca1) + "salgados")
print("Agota com R$" + str(d) + "voce so pode comprar" + str(cg1) + "salgados")

File: larissa016/Questao09.java
import java.util.Scanner;

public class Questao09 {
    public static void main (String [] args ) {
        Scanner s;
        double d;
        double ca;
        double cg;
        double ca1;
        double cg1;
        s = new Scanner (System.in);
        System.out.print("Informe o total de dinheiro que possui:");
        d = s.nextDouble ();
        System.out.print("Quanto custava o salgado?");
        ca = s.nextDouble ();
        System.out.print("Quanto custa um salgado?");
        cg = s.nextDouble();
        ca1 = d / ca;
        cg1 = d / cg;

        System.out.println(ca1);
        System.out.println(cg1);







    }
}
